docs/source/ops.rst

```diff
@@ -11,7 +11,15 @@ torchvision.ops
 
 .. autofunction:: nms
```

Comment:
Additionally, can you send a new PR changing line 9, as now all operators support torchscript.

Reply:
Added in #2343